hal/halx86/generic/usage.c

```diff
@@ -279,6 +279,9 @@ HalpReportResourceUsage(IN PUNICODE_STRING HalName,
     LARGE_INTEGER CurrentSortValue, SortValue;
     DbgPrint("%wZ Detected\n", HalName);
 
+#if defined(SARCH_XBOX)
+    UNREFERENCED_PARAMETER(Port);
+#else
```

Comment:
Hm, I wonder if the normal boot mode is affected (by the serial driver). This code path seems to be triggered only when KD is active.

Reply:
Whats the purpose of this hal change, by the way?

Reply:
@HBelusca it's in commit message

Reply:
> @HBelusca it's in commit message

You see, code comments exist for a reason, too.